<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRohteilkontrolle</class>
 <widget class="QDialog" name="DialogRohteilkontrolle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>598</width>
        <height>198</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton_RK00">
         <property name="minimumSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Icons/Rohteilkontrolle/RohteilKontrolle00_192.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_RK01">
         <property name="minimumSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Icons/Rohteilkontrolle/RohteilKontrolle01_192.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_RK02">
         <property name="minimumSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Icons/Rohteilkontrolle/RohteilKontrolle02_192.PNG</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
